content/learn/migration-guides/0.12-to-0.13.md

```diff
@@ -1153,6 +1153,18 @@ If you were using the pipelined rendering channels, `MainToRenderAppSender` and
 
 Bevy has previously used rendering-specific types like `UVSphere` and `Quad` for primitive mesh shapes. These have now been deprecated to use the geometric primitives newly introduced in version 0.13.
 
+- `bevy_render::mesh::Capsule` is deprecated use `bevy_math::primitives::dim3::Capsule3d` instead;
```

Comment:
Shouldn't this also mention `bevy_math::primitives::dim3:Capsule2d` since from my understanding `bevy_render::mesh::Capsule` was dimension-agnostic and could work for both 2d and 3d—even if primarily intended for 3d—, and was probably used by someone for 2d? Or is this unwanted since there is already a minor section mentioning that Capsule is now Capsule3d and any 2d uses should use Capsule 2d?

Reply:
The deprecated [Capsule](https://github.com/bevyengine/bevy/blob/release-0.12.1/crates/bevy_render/src/mesh/shape/capsule.rs) was intended for 3d only, was we can see on [`Into<Mesh>` impl](https://github.com/bevyengine/bevy/blob/22e39c4abf6e2fdf99ba0820b3c35db73be71347/crates/bevy_render/src/mesh/shape/capsule.rs#L47)